integration-tests/src/test/java/org/web3j/protocol/besu/BesuPrivacyQuickstartIntegrationTest.java

```diff
@@ -65,32 +61,47 @@ public class BesuPrivacyQuickstartIntegrationTest {
             Credentials.create("ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f");
 
     private static final Base64String ENCLAVE_KEY_ALICE =
-            Base64String.wrap("A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=");
+            Base64String.wrap("BULeR8JyUWhiuuCMU/HLA0Q5pzkYT+cHII3ZKBey3Bo=");
     private static final Base64String ENCLAVE_KEY_BOB =
-            Base64String.wrap("Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs=");
+            Base64String.wrap("QfeDAys9MPDs2XHExtc84jKGHxZg/aj52DTh0vtA3Xc=");
     private static final Base64String ENCLAVE_KEY_CHARLIE =
-            Base64String.wrap("k2zXEin4Ip/qBGlRkJejnGWdP9cjkK+DAvKNW31L2C8=");
+            Base64String.wrap("1iTZde/ndBHvzhcl7V68x44Vx7pl8nwx9LqnM/AfJUg=");
 
     private static final BesuPrivacyGasProvider ZERO_GAS_PROVIDER =
             new BesuPrivacyGasProvider(BigInteger.valueOf(0));
 
     private static Besu nodeAlice;
     private static Besu nodeBob;
     private static Besu nodeCharlie;
+    private static Besu rpcNode;
 
     @BeforeAll
-    public static void setUpOnce() {
-        nodeAlice = Besu.build(new HttpService("http://localhost:20000"));
-        nodeBob = Besu.build(new HttpService("http://localhost:20002"));
-        nodeCharlie = Besu.build(new HttpService("http://localhost:20004"));
+    public static void setUpOnce() throws InterruptedException, IOException {
+        runScript("src/test/resources/quorum-test-network/run.sh");
+
+        nodeAlice = Besu.build(new HttpService("http://127.0.0.1:20000"));
+        nodeBob = Besu.build(new HttpService("http://127.0.0.1:20002"));
+        nodeCharlie = Besu.build(new HttpService("http://127.0.0.1:20004"));
+        rpcNode = Besu.build(new HttpService("http://127.0.0.1:8545"));
+
+        int blockNumber = 0;
+        do {
+            TimeUnit.MINUTES.sleep(1);
+            blockNumber = rpcNode.ethBlockNumber().send().getBlockNumber().intValue();
+        } while (blockNumber <= 100);
+    }
+
+    @AfterAll
+    public static void stopNetwork() {
+        runScript("src/test/resources/quorum-test-network/remove.sh");
     }
 
     @Test
     public void testConnection() throws IOException {
-        assertEquals(nodeAlice.web3ClientVersion().send().getWeb3ClientVersion(), (CLIENT_VERSION));
-        assertEquals(nodeBob.web3ClientVersion().send().getWeb3ClientVersion(), (CLIENT_VERSION));
-        assertEquals(
-                nodeCharlie.web3ClientVersion().send().getWeb3ClientVersion(), (CLIENT_VERSION));
+        assertNotNull(rpcNode.web3ClientVersion().send().getWeb3ClientVersion());
+        assertNotNull(nodeAlice.web3ClientVersion().send().getWeb3ClientVersion());
+        assertNotNull(nodeBob.web3ClientVersion().send().getWeb3ClientVersion());
+        assertNotNull(nodeCharlie.web3ClientVersion().send().getWeb3ClientVersion());
```

Comment:
it would be nice to assert also the chain ID

Reply:
okay will add assert for chain ID too